spec/views/teams/index.html.erb_spec.rb

```diff
@@ -39,4 +39,22 @@
     render
     expect(rendered).to have_selector(:link_or_button, t('teams.index.create_team'))
   end
+
+  it "renders the delete button when signed in and you are an admin" do
```

Comment:
@lheilmann we think this test doesn't belong here... It probably got in accidentally when resolving merge conflicts.

Reply:
@birneamstiel You're right! Should I exlude it from this PR?